tools/install_so_files.sh

```diff
@@ -1,7 +1,7 @@
 set -e -x
 
 if [ "$TF_NEED_CUDA" == "1" ]; then
-  CUDA_FLAG="--crosstool_top=//build_deps/toolchains/gcc7_manylinux2010-nvcc-cuda11:toolchain"
+  CUDA_FLAG="--crosstool_top=@ubuntu20.04-gcc9_manylinux2014-cuda11.2-cudnn8.1-tensorrt7.2_config_cuda//crosstool:toolchain"
```

Comment:
What we want to do with the new manylinux2014  toolchains? 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/toolchains/remote_config/configs.bzl

Reply:
https://github.com/tensorflow/tensorflow/blob/master/.bazelrc#L608-L619

Reply:
@angerson @perfinion  What we need to switch from the custom_ops repo toolchain to `@ubuntu20.04-gcc9_manylinux2014-cuda11.2-cudnn8.1-tensorrt7.2_config_cuda//crosstool:toolchain` ?

Reply:
As custom the original toolchain was just cloned from TF custom ops repo, now that it is unmaintained, I suppose we need to import all the TF workspaces [like in TF IO](https://github.com/tensorflow/io/blob/master/WORKSPACE#L115-L144) and make some other refactoring to remove our dependency from the custom ops repo infrastructure.

Reply:
GPU tests passing using the remote toolchains

Reply:
So it Is what more or less whatintroduced here in the last days.

 But just for the record:
 now users are implcitly going to download the Tensorflow tarball, llvm tarball etc..

Reply:
Just during build, not for the packaged whl. Totally fine with that given we're loosing ~1900 lines of tech debt

Reply:
Yes on build and also to contribute a PR we download many things. This is a SIG-build techincal debt with SIGs that are custom ops "orphan".